Add test steps into README

yuxisun1217 opened the pull request:
Signed-off-by: Yuxin Sun <yuxisun@redhat.com>

teg reviewed (commented):
Thanks for this @yuxisun1217! This looks great! I'll let @Gundersanne have a look before merging, but looks good to me!

Gundersanne commented on README.md:
```diff
@@ -3,5 +3,68 @@
-Coverage report is available from
+### Preparation
+1. Clone the following repositories:
+    * https://github.com/osbuild/image-builder-frontend
```
I think we can leave the image-builder-frontend and insights-proxy setup out of the image-builder readme.

Gundersanne commented on README.md:
```diff
@@ -3,5 +3,68 @@
+    * https://github.com/osbuild/image-builder
+    * https://github.com/RedHatInsights/insights-proxy
+
+2. Setting up the proxy
```
See above, the proxy is not required to run the integration tests.

Gundersanne commented on README.md:
````diff
@@ -3,5 +3,68 @@
+    ```
+   
+### CI
+* https://osbuildci.cloud.paas.psi.redhat.com/me/my-views/view/all/job/osbuild/job/image-builder/job/main/
````
This is an internal url, personally don't think it should be in the public readme.

Gundersanne reviewed (changes requested):
Thanks! Looks good, some minor change requests.

yuxisun1217 commented:
Thank @Gundersanne ! I removed the image-builder-frontend and insights-proxy repos, removed CI section, and cut some redundant words. Please help to review again.

yuxisun1217 commented:
BTW, what's our custom here if I want to change code in a PR? Force push to the same commit or create a new commit into the same branch? I used to force push before. Thanks!

Gundersanne reviewed (approved):
Nice, thank you

Force pushing is best, alternatively adding commits on top and then using github to squash is also fine (though that only works when there needs to be only 1 commit merged in the end).
